corehq/apps/cloudcare/static/cloudcare/js/form_entry/form_ui.js

```diff
@@ -635,7 +638,7 @@ hqDefine("cloudcare/js/form_entry/form_ui", function () {
 
         self.hasAnyNestedQuestions = function () {
             return _.any(self.children(), function (d) {
-                if (d.type() === constants.QUESTION_TYPE || d.type() === constants.REPEAT_TYPE || d.type() === constants.GROUPED_QUESTION_TILE_ROW_TYPE) {
+                if (d.type() === constants.QUESTION_TYPE || d.type() === constants.REPEAT_TYPE || d.type() === constants.GROUPED_ELEMENT_TILE_ROW_TYPE) {
```

Comment:
You'll need to check nested elements for `GROUPED_ELEMENT_TILE_ROW_TYPE` since it can contain groups

Reply:
https://github.com/dimagi/commcare-hq/pull/33834/commits/b5037b04ca9a2bafe53850bf8b53622f1e18de20